index.html

```diff
@@ -2421,6 +2434,11 @@ <h3>Processing Capabilities</h3>
     indicating the session will not implicitly trust untrusted
     or self-signed TLS certificates on <a data-lt=go>navigation</a>.
 
+   <dt>"<code>strictFileInteractability</code>"
+    <dd><a>Boolean</a> initially set to <code>false</code>,
+    indicating that interactabilty checks will be applied to
+    &lt;input type=file&gt;.
```

Comment:
This should probably  read like 
```suggestion
    an <a><code>input</code> element</a> whose <a><code>type</code> attribute</a> is in the <a>File Upload state</a>
```

Reply:
Is that clearer? This text is informative, so it doesn't have to be precise.

Reply:
It still uses `&lt;input type=file&gt;` when it should say `an <a><code>input</code> element</a> in the <a>file upload state</a>`.

Reply:
Right. AFAICT this text is all informative and we could remove it without changing behaviour. In light of that I claim that `<input type=file>` is easier to understand.

Reply:
Oh I see what you mean this is descriptive, resolving this conversation.